### DISCUSSION TITLE
Please add Livewire v2.0 support

### DISCUSSION THREAD

**Discussion by vishalinfyom** [opcodesio/log-viewer#170] [Ideas]
Will you please add support for livewire v2.0

**Comment by arukompas** [comment]
Hey @vishalinfyom , this package already uses Livewire 2.x. Do you need support for the exact 2.0 version? Why so? It's always a good idea to update your dependencies in order to have bug fixes and new features. Are you able to upgrade to the latest Livewire 2.x version before installing the Log Viewer?
